assets/js/components/settings/SettingsActiveModule/Header.js

```diff
@@ -276,13 +281,23 @@ export default function Header( { slug } ) {
 							slug === 'analytics' &&
 							! hasAnalyticsAccess &&
 							! isGA4Connected && (
-								<p className="googlesitekit-settings-module__status">
-									{ __(
-										'Google Analytics 4 is not connected',
-										'google-site-kit'
+								<Fragment>
+									{ hasResolvedAnalyticsAccess ? (
```

Comment:
Using a ternary here because having two different blocks for `hasResolvedAnalyticsAccess` and `! hasResolvedAnalyticsAccess` causes an ESLint complexity error.

Now that this component has more than a few statuses/actions displayed conditionally, I think the conditions should be handled a bit more gracefully in a better readable manner, potentially in a new issue.

Reply:
Understood, thank you for the clarification, @aaemnnosttv! I believe #2280 was the issue. I've updated this entire module status composition in a set of if-else blocks, which **I think** is more readable and sequential. Please let me know what you think and I'm happy to change it otherwise.

https://github.com/google/site-kit-wp/blob/3b0903a79ef6aee33624d3121123f69a2661f1c3/assets/js/components/settings/SettingsActiveModule/Header.js#L192-L252